config/environments/development.rb

```diff
@@ -37,13 +37,12 @@
     password: Rails.application.secrets.email_provider_password
   }
   # ActionMailer Config
-  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.